Gemfile

```
@@ -7,6 +7,10 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 gem 'rake'
 gem 'dotenv', '~> 2.1'
 
+group :package do
+  gem 'ood_packaging', '0.1.0'
```

```suggestion
  gem 'ood_packaging', '~> 0.1'
```

Do you think we'll have this compatibility?

---

I think this will be fine. My goal is to not bump to 1.0.0 until we begin building OnDemand 2.2 or whatever comes after 2.1 as that would be when things like Ruby or NodeJS versions might change or other things that would require major bump in version for packaging tools.